README.md

```diff
@@ -88,7 +88,7 @@ You can see your node on [telemetry](https://telemetry.polkadot.io/#list/0x34f61
 ## Contributing
 
 ### Roadmap
-[Roadmap](docs/ROADMAP.md)
+[Roadmap](https://itering.notion.site/9617e154ec884b07a7cee9a056374e42?v=0c3e4d9f257646c486a32a0425ee3a93)
```

Comment:
![image](https://user-images.githubusercontent.com/11801722/159483421-1a80def4-5b0c-48e7-a6bb-c00178105f8d.png)

Perhaps we should update tasks' latest status, by the way, some of them have finished.